mllib/src/main/scala/org/apache/spark/mllib/classification/LogisticRegression.scala

```diff
@@ -17,14 +17,17 @@
 
 package org.apache.spark.mllib.classification
 
+import org.apache.spark.SparkContext
 import org.apache.spark.annotation.Experimental
+import org.apache.spark.mllib.classification.impl.GLMClassificationModel
 import org.apache.spark.mllib.linalg.BLAS.dot
 import org.apache.spark.mllib.linalg.{DenseVector, Vector}
 import org.apache.spark.mllib.optimization._
 import org.apache.spark.mllib.regression._
-import org.apache.spark.mllib.util.{DataValidators, MLUtils}
+import org.apache.spark.mllib.util.{DataValidators, Exportable, Importable}
```

Comment:
About the names, we have `Exportable` with `save()` and `Importable` with `load()`. I have two questions:

1) Would `Saveable` and `save` be a better match? I'm looking at the search results from grepcode:

* `Saveable` as an interface: http://grepcode.com/search/?start=0&query=Saveable&entity=type&k=i
* `Exportable` as an interface: http://grepcode.com/search?query=Exportable&start=0&entity=type&n=&k=i

People use both but I didn't see a combination of `Exportable` and `save()`. I hope `savable` or `saveable` is still a valid word. Same applies to `Exportable`/`Loadable` and `load()`.

2) If an instance is `Importable`, it means we can import it from somewhere. This is not true for `object Model`. We don't import `object Model` but we use `object Model` to import `class Model`. Should the interface be called `Importer`/`Loader` instead?